internal/gojsonschema/schema_test.go

```diff
@@ -23,6 +23,7 @@
 //
 // created          16-06-2013
 
+// nolint: deadcode // Package in development (2021).
```

Comment:
Hmm is it? We've vendored it because we needed something changed. I guess in a follow-up, we could drop the dead code instead. (This PR is large enough, feel free to ignore this comment :D)

Reply:
Ahh, roger. I just assumed since there's been a commit to the file in the past year.